### PR TITLE
Team victorious secret Final blog

### PR DESCRIPTION
**Pull request by saumya1906**
Sorry for overshooting the deadline.

### CODE REVIEW

File: blog/_posts/2017-09-12-Final_blog_post.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: “”
```
Comment (nerdyvaishali) [inline blog/_posts/2017-09-12-Final_blog_post.md]: Hi, thanks for sending your 2nd blog post PR. Please check out the comments below.

File: blog/_posts/2017-09-12-Final_blog_post.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (v-thakkar) [inline blog/_posts/2017-09-12-Final_blog_post.md]: Hi, thanks for sending your 2nd blog post PR. Please check out the comments below.

File: blog/_posts/2017-09-12-Final_blog_post.md (hunk 1 above)
Comment (v-thakkar) [inline blog/_posts/2017-09-12-Final_blog_post.md]: Title of the blog post is missing here. Can you please add it?

File: blog/_posts/2017-09-12-Final_blog_post.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+title: “”
+layout: post
+created_at: Tue Sept 12 2017
+permalink: blog/2017-09-06-final-blog-post-team-victorious-secret
```
Comment (v-thakkar) [inline blog/_posts/2017-09-12-Final_blog_post.md]: Permalink and file name doesn't match. Please fix the date and name both. Fix a date everywhere with Wed Sept 13 2017.

File: blog/_posts/2017-09-12-Final_blog_post.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+current: blog
+author: Katyayani Singh and Saumya Balodi
+twitter: VictoriuSecret
+categories:
+- student-posts
+---
+
+![Team Victorious Secret](/img/blog/2017/team_victorious_secret1.jpeg)<font color="grey"><small><i>Picture Credits: Saumya</i></small></font>
```
Comment (v-thakkar) [inline blog/_posts/2017-09-12-Final_blog_post.md]: Can you please use image-credits class here? Check out other blog posts for the reference.

File: blog/_posts/2017-09-12-Final_blog_post.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+* We finally started understanding how JavaScript actually works. It is one thing to be able to put together code to make something work, and it is another thing altogether to actually know exactly why it works. 
+* We were introduced to React. This was the most exciting bit of our summer. Working with something as popular and novel as React was an extraordinary experience.
+* We realized how designers and developers collaborate to build seamless user experiences. Working with the p5.js web editor made us aware of how much work goes into designing the user flow of an application even before the developers start coding. A lot of thinking and rethinking goes into user interactions and the response of the application. A shout out to the brilliant designers and developers of the web editor! Working with you, and learning from your work was truly amazing.
+* A tiny push in times of need can go a long way. Our mentor Cassie and our supervisor Pilar, have been our pillars of strength and support in this entire journey. It is thanks to their help, that we were able pull through testing times.
```
Comment (v-thakkar) [inline blog/_posts/2017-09-12-Final_blog_post.md]: Add missing to: 'we were able **to** pull'

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Review by v-thakkar — APPROVED** [review]

**Review by v-thakkar — CHANGES_REQUESTED** [review]
Hi, thanks for the quick fixes @saumya1906  . For some reason, image is broken. Can you please fix that? We will be ready to publish the blog after that. Thanks. :)

**Review by v-thakkar — APPROVED** [review]

**Pull request merged**